modules/cli/src/main/scala/coursier/cli/install/InstallOptions.scala

```diff
@@ -24,8 +24,6 @@ final case class InstallOptions(
   @Recurse
     envOptions: EnvOptions = EnvOptions(),
 
-  addChannel: List[String] = Nil,
```

Comment:
See the comment above. Let's not remove options for now, but rather print a deprecation warning.